### PR TITLE
Purch Ad Support for Amp-Ads

### PR DESCRIPTION
**Pull request by jsunpurch**
Please review this pul request for adding Purch Ads Support on amp-ads

### CODE REVIEW

**Comment by jsunpurch** [comment]
The CLA is under the company Purch.

File: ads/purch.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function purch(global, data) {
+  validateData(data, [],
+      ['pid', 'divid']);
+  global.data = data;
+
+  const adsrc = 'https://ramp.purch.com/serve/creative_amp.js';
+  if (typeof adsrc != 'undefined') {
```
Comment (lannka) [inline ads/purch.js]: why is this check?

File: ads/purch.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    validateSrcPrefix('https:', adsrc);
+    writeScript(global, adsrc);
+  }
+}
```
Comment (lannka) [inline ads/purch.js]: need an extra line at the end

**Comment by jsunpurch** [comment]
@lannka  - I made the changes requested to add in ability for renderStart. 

@googlebot - the CLA is signed under the company Purch. My email should be in the google group 'purch-google-contributors'.

File: ads/purch.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  data-pid="2882"
+  data-divid="rightcol_top">
+</amp-ad>
+```
+
+## Configuration
+
+Supported parameters:
+
+- data-pid    placement id
```
Comment (zhouyx) [inline ads/purch.md]: i'd suggest use code style and use`data-pid` and `data-divid` instead.

**Comment by zhouyx** [comment]
pull request looks good. one last request from me.
ping @lannka for a final look.

File: ads/purch.md (hunk 3 above)
Comment (jsunpurch) [inline ads/purch.md]: @zhouyx  - I updated the code to reflect your review.

**Comment by zhouyx** [comment]
@jsunpurch I didn't find Purch in our CLA list. 
>>>
CLA: for anyone who has trouble to pass the automatic CLA check in a pull request, try to follow the guidelines provided by the CLA Bot. Common mistakes are 1) used a different email address in git commit; 2) didn't provide the exact company name in the PR thread.
>>>

**Comment by isyndicate** [comment]
@zhouyx we have a confirmation document from DocuSign (that was signed 10/13). Can you please double-check? Corporation name is "Purch".

**Comment by zhouyx** [comment]
@isyndicate thanks for the update. According to [https://cla.developers.google.com/about] I assume it's still under process.
>>>
After the agreement is electronically signed, it will need to be reviewed by someone at Google. We check to make sure that there isn't already an agreement on file for your company and that it was signed by someone with necessary authority. This normally takes a few days. Once it's been accepted, you will receive an email and it will appear in the "Agreements you are covered by" section of the Agreements page.
>>>

**Comment by jsunpurch** [comment]
@zhouyx  - Can you check again? We received an email that the CLA is completed.

**Comment by zhouyx** [comment]
@jsunpurch  Thanks for the Pull Request! Merged.

**Pull request merged**